www/_posts/2016-02-09-plugins-release.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+author:
+    name: Vladimir Kotikov
+    url: https://github.com/vladimir-kotikov
+title:  "Plugins Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today:
+
+* cordova-plugin file@4.1.1
+* cordova-plugin-inappbrowser@1.3.0
+* cordova-plugin-media@2.2.0
+* cordova-plugin-statusbar@2.1.1
+* cordova-plugin-spashscreen@3.2.0
+* cordova-plugin-wkwebviewengine@1.0.2
+
+----
+You can update any plugin by removing it, and then re-adding it.
+
+ E.g. To update your media plugin:
+
+    cordova plugin rm cordova-plugin-media --save
+    cordova plugin add cordova-plugin-media --save
```

Comment:
I would suggest adding the plugin version to the add command. `cordova plugin add cordova-plugin-media@2.2.0 --save`.